src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -215,7 +238,11 @@
                                                                 3000)
                                                  (check-qr-code-data data)))
             scan-qr-code-tab?                (= @active-tab 1)
-            show-camera?                     (and scan-qr-code-tab? @camera-permission-granted?)
+            show-camera?                     (and scan-qr-code-tab?
+                                                  (if platform/ios?
```

Comment:
why the platform check?  `@preflight-check-passed?` is true on android anyway -
this could just be 

```
show-camera? (and scan-qr-code-tab? 
       @camera-permission-granted?
       @preflight-check-passed?)
       ```

Reply:
Yes, It's true on Android. I forgot to update the checks inside the render.

It's updated now. 👍